control/iosys.py

```diff
@@ -959,10 +956,18 @@ def __init__(self, syslist, connections=[], inplist=[], outlist=[],
                 self.input_map[self._parse_input_spec(spec), index] = 1
 
         # Convert the output list to a matrix: maps subsystems to system
+        # why can subsystem inputs map to outputs? shouldn't it be
+        # system inputs (hence self.ninputs instead of ninputs)?
         self.output_map = np.zeros((self.noutputs, noutputs + ninputs))
```

Comment:
```suggestion
        self.output_map = np.zeros((self.noutputs, noutputs + ninputs))
```
`(self.noutputs, noutputs + ninputs)` should be correct. See: `set_output_map()`:
https://github.com/python-control/python-control/blob/b68872927b79cf5c10dbf7ae27f93affbefb5ffe/control/iosys.py#L1291-L1315

This also made me realize, that  `inputs` and `outputs` are missing in the `Parameters` section of the docstring for `InterconnectedSystem`.